The code example is possibly outdated

I get a type error for this example:
```ts
import * as aws from "@pulumi/aws";
import * as awsx from "@pulumi/awsx";

const logGroup = new aws.cloudwatch.LogGroup("nginxLogs", {
    retentionInDays: 7,
});
const nginx = new awsx.ecs.FargateService("nginx", {
    taskDefinitionArgs: {
        logGroup, // use our custom LogGroup
        containers: {
            nginx: {
                image: "nginx",
                memory: 128,
            },
        },
    },
});
```
The error:
```ts
Type 'LogGroup' has no properties in common with type 'DefaultLogGroupArgs'.ts(2559)
```

File: [themes/default/content/docs/clouds/aws/guides/cloudwatch.md](https://www.pulumi.com/docs/clouds/aws/guides/cloudwatch/)

Confirming that I'm able to reproduce the error reported locally. Fix in progress.


```bash
❯ pulumi up          
Please choose a stack, or create a new one: <create a new stack>
Please enter your desired stack name.
To create a stack in an organization, use the format <org-name>/<stack-name> (e.g. `acmecorp/dev`): test 
Please enter your desired stack name.
Created stack 'test'
Previewing update (test)

View in Browser (Ctrl+O): https://app.pulumi.com/diana-pulumi-corp/pulumi-hugo_issues_3446/test/previews/56446d44-dc41-4d03-9d54-afdc440f6e1e

     Type                 Name                          Plan       Info
 +   pulumi:pulumi:Stack  pulumi-hugo_issues_3446-test  create     1 error

Diagnostics:
  pulumi:pulumi:Stack (pulumi-hugo_issues_3446-test):
    error: Running program '/Users/d/github/desteves/TEST-ISSUES/pulumi-hugo_issues_3446/index.ts' failed with an unhandled exception:
    TSError: ⨯ Unable to compile TypeScript:
    index.ts(9,9): error TS2559: Type 'LogGroup' has no properties in common with type 'DefaultLogGroupArgs'.
    index.ts(11,13): error TS2741: Property 'name' is missing in type '{ image: string; memory: number; }' but required in type 'TaskDefinitionContainerDefinitionArgs'.
```



Other files:

`index.ts`
```typescript
import * as aws from "@pulumi/aws";
import * as awsx from "@pulumi/awsx";

const logGroup = new aws.cloudwatch.LogGroup("nginxLogs", {
    retentionInDays: 7,
});
const nginx = new awsx.ecs.FargateService("nginx", {
    taskDefinitionArgs: {
        logGroup, // use our custom LogGroup
        containers: {
            nginx: {
                image: "nginx",
                memory: 128,
            },
        },
    },
});
```

`package.json`
```json
{
    "name": "pulumi-hugo_issues_3446",
    "main": "index.ts",
    "devDependencies": {
        "@types/node": "^16"
    },
    "dependencies": {
        "@pulumi/pulumi": "^3.0.0",
        "@pulumi/aws": "^5.0.0",
        "@pulumi/awsx": "^1.0.0"
    }
}
```


`tsconfig.json`
```json
{
    "compilerOptions": {
        "strict": true,
        "outDir": "bin",
        "target": "es2016",
        "module": "commonjs",
        "moduleResolution": "node",
        "sourceMap": true,
        "experimentalDecorators": true,
        "pretty": true,
        "noFallthroughCasesInSwitch": true,
        "noImplicitReturns": true,
        "forceConsistentCasingInFileNames": true
    },
    "files": [
        "index.ts"
    ]
}
```

--- a/themes/default/content/docs/clouds/aws/guides/cloudwatch.md
+++ b/themes/default/content/docs/clouds/aws/guides/cloudwatch.md
@@ -92,9 +92,12 @@ const logGroup = new aws.cloudwatch.LogGroup("nginxLogs", {
 });
 const nginx = new awsx.ecs.FargateService("nginx", {
     taskDefinitionArgs: {
-        logGroup, // use our custom LogGroup
+        logGroup: {
+            existing: logGroup // use our custom LogGroup
+        },
         containers: {
             nginx: {
+                name: "nginx",
                 image: "nginx",
                 memory: 128,
             },

